### PR TITLE
Filter component

### PR DESCRIPTION
**Pull request by chandelieraxel**
Pr to add the component who show the button with pre parameters filters

### CODE REVIEW

File: src/Components/filterBar.js [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
-const FilterBar = () => (
+const mapStateToProps = state => ({
+  filterKeywordPageDisplay: state.filterKeywordPageDisplay,
+});
+
+const FilterBar = ({ dispatch, filterKeywordPageDisplay }) => (
   <div>
     <div className="footer-box">
-      <LogoFilter className="filter-logo" />
+      <LogoFilter className="filter-logo" onClick={() => dispatch({ type: 'DISPLAY_FILTER_PAGE', filterKeywordPageDisplay: !filterKeywordPageDisplay })} />
```
Comment (erabin05) [inline src/Components/filterBar.js]: You should change the ```filterKeywordPageDisplay``` on ```!filterKeywordPageDisplay``` in the reducer

File: src/reducers/poisReducer.js [hunk 2]
```diff
@@ -37,16 +38,26 @@ const poisReducer = (state = initialState, action) => {
...
         ...state,
         searchBarValueInput: action.searchBarValueInput,
       };
-    case 'HANDLE_SUBMIT_SEARCHBAR':
+    case 'HANDLE_KEYWORD_FILTERING':
       return {
         ...state,
         filteredPoiByKeyword: action.filteredPoiByKeyword,
       };
+    case 'CLOSE_FILTER_COMPONENT':
```
Comment (erabin05) [inline src/reducers/poisReducer.js:95]: if you close it, it is always change in false ? If yes, you could change it as false directly in the reducer than in the dispatch

**Review by erabin05 — CHANGES_REQUESTED** [review]
- ```CLOSE_FILTER_COMPONENT``` : you could change ```filterKeywordPageDisplay``` in false directly in the reducer
- ```HANDLE_KEYWORD_FILTERING```: change the value in the reducer not in the dispatch

File: src/Components/filterBar.js (hunk 1 above)
Comment (Spartakyste) [reply src/Components/filterBar.js]: Gotcha, changed it to a permanent falsen and suppressed the corresponding actions in the component

File: src/reducers/poisReducer.js (hunk 2 above)
Comment (chandelieraxel) [reply src/reducers/poisReducer.js:95]: Gotcha, changed it to a permanent falsen and suppressed the corresponding actions in the component

File: src/Components/filterBar.js (hunk 1 above)
Comment (chandelieraxel) [reply src/Components/filterBar.js]: It's corrected, deleted the new dead code in the corresponding component aswell

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**